backend/Recodo.BLL/Services/TeamService.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Recodo.DAL.Context;
+using Recodo.DAL.Entities;
+using System.Threading.Tasks;
+
+namespace Recodo.BLL.Services
+{
+    public class TeamService
+    {
+        private readonly RecodoDbContext _context;
+
+        public TeamService(RecodoDbContext context)
+        {
+            _context = context;
+        }
+
+        public async Task CreateTeam(User userEntity)
+        {
+            Team team = new Team();
+            team.Name = userEntity.Email.Split('@')[0] + "'s team";
```

Comment:
Why not provide the user with a way to set his own name for the team?

Reply:
Now take from Workspace, like loom